recipes/libint2/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -x
+echo $PREFIX
+export CXXFLAGS="${CXXFLAGS} -O2 -pipe -march=x86-64 -std=c++11 -fPIC -fopenmp"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+./configure --prefix=${PREFIX} --enable-shared
+
+make -j4
+make -j4 check
+make -j4 install
```

Comment:
Would replace all of the `-j4`'s with `-j${CPU_COUNT}` as that is set correctly for CIs.